integration/test_indexstats.py

```
@@ -0,0 +1,162 @@
+
+        # Add a few records, then re-run.
+        idx.load_data(client, 3)
+        reply = client.execute_command("FT._DEBUG", "INDEXSTATS", idx.name)
+        self._check_reply(reply, has_records=True)
+
+        # Unknown-attribute error.
+        try:
+            client.execute_command(
+                "FT._DEBUG", "INDEXSTATS", idx.name, "no_such_field"
+            )
+            assert False, "expected error for unknown attribute"
```

Ruff (0.15.12)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)